sqsworkers/crew.py

```diff
@@ -153,7 +187,10 @@ def run(self):
             'connected to sqs, approx. %s msgs on queue' %
             self.queue.attributes['ApproximateNumberOfMessages']
         )
-        self.poll_queue()
+        if self.bulk_mode:
+            self.bulk_poll_queue()
```

Comment:
do we need a separate function? it could be a condition in poll_queue itself right?

Reply:
It's lot cleaner this way. We are not changing existing function at all.

Reply:
👍